test/smart-wallet/refund-estimation.ts

```diff
@@ -446,18 +469,19 @@ describe("Wallet tx gas estimations with and without refunds", function () {
 
     // safeTx.baseGas = handlePaymentEstimate + 17100 - 9794 + 2360 + 5000; // add 5000 offset for delegate call?;
 
-    safeTx.baseGas = handlePaymentEstimate + 17100 - 7306 + 2360 + 5000; // add 5000 offset for delegate call?;
+    console.log(safeTx);
+
+    safeTx.gasPrice = 156849;
+    safeTx.tokenGasPriceFactor = 1000000;
+
+    safeTx.baseGas =
+      21000 + 5000 + handlePaymentEstimate + 17100 - 7306 + 2360 + 5000; // add 5000 offset for delegate call?;
```

Comment:
What is 7306?

Reply:
Events + internal call  2360
handle ETH/native payment estimation is 7306
17100 i believe was for state changes offset as per very old notes and calcs

However gas estimation test cases needs a lot of rework.